tasks/update_appraisal_gemfiles.rake

```diff
@@ -1,68 +0,0 @@
-TRACER_VERSIONS = %w[
-  2.1
-  2.2
-  2.3
-  2.4
-  2.5
-  2.6
-  2.7
-  3.0
-  3.1
-  3.2
-  jruby-9.2.8.0
-  jruby-9.2
-  jruby-9.3
-].freeze
-# ADD NEW RUBIES HERE
-
-FORCE_BUNDLER_VERSION = {
-  '2.3' => '1.17.3', # Some groups require bundler 1.x https://github.com/DataDog/dd-trace-rb/issues/2444
-}.freeze
-
-desc 'Installs gems based on Appraisals and Gemfile changes, ' \
-     'accepts list of tracer versions as task argument, defaults to all versions.'
-task :install_appraisal_gemfiles do |_task, args|
-  tracer_version_arg = args.to_a
-  versions = tracer_version_arg.empty? ? TRACER_VERSIONS : tracer_version_arg
-
-  versions.each do |version|
-    forced_bundler_version = nil
-    bundler_setup = 'gem update bundler'
-
-    if FORCE_BUNDLER_VERSION.include?(version)
-      forced_bundler_version = "_#{FORCE_BUNDLER_VERSION[version]}_"
-      bundler_setup = "gem install bundler -v #{FORCE_BUNDLER_VERSION[version]}"
-    end
-
-    sh [
-         "docker-compose run -e APPRAISAL_GROUP --no-deps --rm tracer-#{version} /bin/bash -c ",
-         "'rm -f Gemfile.lock && #{bundler_setup} && bundle #{forced_bundler_version} install && ",
-         # Appraisal runs `bundle check || bundle install` on `appraisal install`. This skips `bundle install` \
-         # if the `Gemfile.lock` is satisfied by installed gems, even if there are `Gemfile` changes to be processed. \
-         # Adding the `--without` option forces Appraisal to skip `bundle check` and always run `bundle install`. \
-         # `--without` has a small side-effect of getting saving in the local bundler env, but we do not persist \
-         # these changes outside of the current container. \
-         "bundle #{forced_bundler_version} exec appraisal install --without force-appraisal-to-always-run-bundle-install'"
-       ].join
-  end
-end
-
-desc 'Update ALL gems based on Appraisals and Gemfile changes, ' \
-     'accepts list of tracer versions as task argument, defaults to all versions.'
-task :update_appraisal_gemfiles do |_task, args|
-  tracer_version_arg = args.to_a
-  versions = tracer_version_arg.empty? ? TRACER_VERSIONS : tracer_version_arg
```

Comment:
Since we've trained some of our occasional contributors to use these tasks, what do you think of perhaps leaving behind these tasks, but having them only print a message saying "You should use XXX instead. We'll remove this message for dd-trace-rb 2.0"?

That way they don't need to wonder why they've rebased their branch and suddenly this is missing and what they need to do next :)

Reply:
Hmm, the old tasks are kind of broken, maybe having these be shims and either call the new tasks or just fail and print out a message directing them to the new ones?

Re: 2.0, do we consider such dev tasks to be part of the public API?

Reply:
> Hmm, the old tasks are kind of broken, maybe having these be shims and either call the new tasks (with a message, but I fear the message would be lost in the output) or just fail and print out a message directing them to the new ones?

I'm thinking just printing a message saying "we removed this, please use XX instead" -- no need to get fancy.

> Re: 2.0, do we consider such dev tasks to be part of the public API?

Nope! My thinking was that we already have a bunch of things marked as "# remove later when we go 2.0", so if we leave such a note here we'll remember to remove it (and by that time, nobody will be trying to use this).

Reply:
Done